doc/_utils/redirects.py

```diff
@@ -193,11 +193,16 @@
     ("device_guides/nrf70/nrf7002ek_dev_guide", "app_dev/device_guides/nrf70/nrf7002ek_dev_guide"),
     ("device_guides/working_with_nrf/nrf70/nrf7002eb_gs", "app_dev/device_guides/nrf70/nrf7002eb_dev_guide"), # Developing with nRF7002 EB
     ("device_guides/nrf70/nrf7002eb_dev_guide", "app_dev/device_guides/nrf70/nrf7002eb_dev_guide"),
-    ("device_guides/nrf54l", "app_dev/device_guides/nrf54l"), # Developing with nRF54L Series
-    ("device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/working_with_nrf/nrf54l/features"), # Features of the nRF54L15 PDK
-    ("device_guides/working_with_nrf/nrf54l/nrf54l15_gs", "app_dev/device_guides/working_with_nrf/nrf54l/nrf54l15_gs"), # Getting started with nRF54L15 PDK
-    ("device_guides/working_with_nrf/nrf54l/testing_dfu", "app_dev/device_guides/working_with_nrf/nrf54l/testing_dfu"), # Testing the DFU solution
-    ("device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield", "app_dev/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield"), # Developing with Peripheral Sensor node shield
+    ("device_guides/nrf54l", "app_dev/device_guides/nrf54l/nrf54l"), # Developing with nRF54L Series
+    ("app_dev/device_guides/nrf54l", "app_dev/device_guides/nrf54l/nrf54l"),
+    ("device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/nrf54l/features"), # Features of the nRF54L15 PDK
+    ("app_dev/device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/nrf54l/features"),
+    ("device_guides/working_with_nrf/nrf54l/nrf54l15_gs", "app_dev/device_guides/nrf54l/nrf54l"), # Getting started with nRF54L15 PDK
```

Comment:
Perhaps something like removed after 2.8.0 could be added to the comment to make backtracking easier. Example: https://github.com/nrfconnect/sdk-nrf/pull/17282/files#diff-e5180fe49174a7f4386d1131761cd0f4d3e42f7105415f4cdc178454257c884fL85

Reply:
great point, added :)